content/v2.0/reference/flux/stdlib/date/truncate.md

```diff
@@ -55,5 +57,20 @@ date.truncate(t: "2019-06-03T13:59:01.000000000Z", unit: 1m)
 
 date.truncate(t: "2019-06-03T13:59:01.000000000Z", unit: 1h)
 // Returns  2019-06-03T13:00:00.000000000Z
+```
+
+##### Truncate time values using durations
+```js
+import "date"
+
+option now = () => 2020-01-01T00:00:00Z
```

Comment:
It would be good to choose a time like `2020-01-01T00:00:30.500000000Z` so that truncating to the second, minute or hour will actually truncate in the examples below.

Reply:
Good call. I'll do that.